<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmationDialogWindow</class>
 <widget class="QDialog" name="ConfirmationDialogWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Delete Confirmation</string>
  </property>
  <widget class="QPushButton" name="yesButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="noButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="messageTextBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
